cmd/execute.go

```diff
@@ -186,13 +188,30 @@ func watchEthStorage(w watcher.IStorageWatcher, wg *syn.WaitGroup) {
 	defer wg.Done()
 	// Execute over the StorageTransformerInitializer set using the storage watcher
 	LogWithCommand.Info("executing storage transformers")
-	ticker := time.NewTicker(pollingInterval)
-	defer ticker.Stop()
-	for range ticker.C {
-		errs := make(chan error)
-		diffs := make(chan storageUtils.StorageDiff)
-		w.Execute(diffs, errs, queueRecheckInterval)
+	on := viper.GetBool("storageBackFill.on")
+	if on {
+		backFillStorage(w)
+	}
+	w.Execute(queueRecheckInterval, on)
+}
+
+func backFillStorage(w watcher.IStorageWatcher) {
```

Comment:
What happens if the client doesn't support backfill, or the data is beyond the pruning limit and isn't an archive node? 

Just wanna make sure we have the proper error handling present.

Reply:
Well we have to explicitly turn the backfill process on, and as is we have to explicitly configure it with the path for a client that supports it although Rob would prefer we assume it is the same client. If the client doesn't support backfill then it will return errs, the backfill will not occur but the other processes will progress normally. I will add an additional guard to the `StateDiffFetcher` so that it makes a single call to test the endpoint before progressing further.